com.unity.ml-agents/CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ and this project adheres to
  camera frame. The scene is able to train with the provided default config file. (#4511)
 ### Bug Fixes
 #### com.unity.ml-agents (C#)
+ - Fixed the issue of leaking AcademyFixedUpdateSteppers when accessing Academy
+ in editor mode. (#4521)
```

Comment:
```suggestion
 - Fixed a bug where accessing the Academy outside of play mode would cause the Academy to get stepped multiple times when in play mode. (#4532)
```
Changed the wording a bit, and the made it refer to this PR (not the original issue). Feel free to tweak the wording if you want.